pages/common/ts-node.md

```diff
@@ -0,0 +1,28 @@
+# ts-node
+
+> Run TypeScript code directly, without any compiling.
+> More information: <https://www.npmjs.com/package/ts-node>.
+
+- Execute a TypeScript file without compiling (`node` + `tsc`).
+
+`ts-node {{path/to/file.ts}}`
+
+- Execute a TypeScript file without loading `tsconfig.json`:
+
+`ts-node --skip-project {{path/to/file.ts}}`
+
+- Display TS-Node help:
+
+`ts-node --help`
+
+- Evaluate TypeScript code directly on the terminal:
```

Comment:
Perhaps this will read better?
```suggestion
- Evaluate TypeScript code passed as a literal on the command-line:
```

Reply:
@bl-ue Hunspell prefers `command line`.

Reply:
Well, Hunspell doesn't necessarily need to be honored always.  In fact, there are a great many words in all of the English pages that Hunspell doesn't like, but are, in fact, valid.

#5848